### ISSUE TITLE
Please, support Firefox 57.

### ISSUE THREAD

**Issue by apometron** [petea/sage#154]
I love the new Firefox, the Developer Edition. =D

Hey, you should update to the new Firefox.

I am waiting with anxiety.

Cheers,
RodrigoCesarBanhara
https://about.me/apometron

**Comment by Machou** [comment]
Yes, please update :P

**Comment by pawelklimiuk** [comment]
Yes, please update, it's the best rss reader for firefox :P

**Comment by Nolff** [comment]
New (non-beta) Firefox shipped. Sage is now completely broken.

**Comment by dannycolin** [comment]
@Nolff It isn't broken. It's just not compatible with FF57+. If you really need your extensions you can use FF52 it's an Extended Support Release so there's still security patches released for it.

**Comment by apometron** [comment]
But Firefox 52 is insecure. All the extensions share the same space. Firefox 58 Developer Edition is the better Firefox that I used !! =D Please, update. =) You will not update? =((( Are you leaving us? Please, do not do that. Please, mister, update... And do not forget the Developer Edition. =) I use the Developer Edition exclusively !!! =DD Thanks !!!!

**Comment by dotnetjunkie** [comment]
Does it help if we support you using donations?

**Comment by dannycolin** [comment]
@apometron having fun trolling on the internet?

@dotnetjunkie First step, Would be to know if it is possible to have the same features with the new Webextension API.

If anyone is interesed in that [here](https://wiki.mozilla.org/WebExtensions) is the documentation about the new API.

**Comment by apometron** [comment]
@dannycolin I am talking seriously!!! =((((((((((((((((((((((( Are You teasing me? Han?

**Comment by apometron** [comment]
https://cyberfox.8pecxstudios.com/ is in the version 52, it can run the Sage extension. I will test it now!!! =DD

**Comment by apometron** [comment]
CyberFox has a bug with Classic Theme, so it seems it is better stick with WaterFox: https://www.waterfoxproject.org/

**Comment by apometron** [comment]
Yes, WaterFox it is only Firefox fork which Sage works, but it would be better working on Firefox 57. =)

**Comment by GHDeepUser** [comment]
It's the best RSS addon for firefox ,Yes, please update

**Comment by apometron** [comment]
And when WaterFox reach version 57? Where to go for support to Sage?

Support for Firefox Quantum would be nice.

Sage requires at least Firefox 38, this means Palemoon cant run Sage.

http://www.palemoon.org/

This leaves us the Cyberfox (buggy software) and Waterfox.

https://cyberfox.8pecxstudios.com/

https://www.waterfoxproject.org/

Currently, WF is on version 56.

The future is uncertain for Sage when WF go 57.

Because of this we need an update.

Please, sir developer, do not leaves us without an update.

**Comment by dotnetjunkie** [comment]
What alternatives for Sage are there? Any tool or add-on that comes close to Sage?

**Comment by saffroy** [comment]
@dotnetjunkie I switched from Sage to Brief, which has a similar UI and works with FF57:
https://addons.mozilla.org/en-US/firefox/addon/brief/

I suggest you export your RSS feeds to an OPML file from Sage before switching.

**Comment by Machou** [comment]
I switched to Feedbro
https://addons.mozilla.org/fr/firefox/addon/feedbroreader/

But i want Sage ! :(

**Comment by Machou** [comment]
### THE UPDATE WE WANT §

@petea can you update Sage ! please !

**Every day is a great sadness for me, because for months I have been living without this addon. _lol_.**

Come on! Come on! Update it for your great community, **you'll make millions of people happy.**

PLEASE, PLEASE, PLEASE; or please, please, but just : PLEASE ! :D

**Comment by dauphine-dev** [comment]
I was missing Sage then I made a clone. It is maned Drop Feeds. It is available here :

https://addons.mozilla.org/fr/firefox/addon/drop-feeds/

_@petea : If you don't want this message (it is like a ads...) then please tell me, I will delete it._

**Comment by GHDeepUser** [comment]
> I was missing Sage then I made a clone. It is maned Drop Feeds. It is available here :
> 
> https://addons.mozilla.org/fr/firefox/addon/drop-feeds/
> 
> _@petea : If you don't want this message (it is like a ads...) then please tell me, I will delete it._

@dauphine-dev  Thanks, it's very useful, I stared. Do you have a plan on Chrome platform? use chrome more often than  firefox😲

**Comment by dauphine-dev** [comment]
@ysjlion 
No plan to port Drop Feeds on Chrome platform in the short term. But yes at long term. But I have to analyze first what it is needed to do that, and I will see if I can/want/will do a port for Chrome platform. I think I will look at that in 2020.
